backend/src/tools/chunk_documents.py

```diff
@@ -0,0 +1,29 @@
+from langchain.text_splitter import RecursiveCharacterTextSplitter
+from langchain.docstore.document import Document as LangchainDocument
```

Comment:
Any reason you are importing as LangchainDocument?

Reply:
Fixed now, its `Document` everywhere